source/macos/AdaptiveCards/AdaptiveCards/AdaptiveCard.swift

```diff
@@ -110,3 +110,33 @@ public struct HyperlinkColorConfig {
         self.underlineStyle = underlineStyle
     }
 }
+public struct ChoiceSetButtonConfig {
+    public static let `default` = ChoiceSetButtonConfig(checkOnIcon: BundleUtils.getImage("checkOn", ofType: "png"),
+                                                        checkOffIcon: BundleUtils.getImage("checkOff", ofType: "png"),
```

Comment:
```suggestion
    public static let `default` = ChoiceSetButtonConfig(checkOnIcon: nil,
                                                        checkOffIcon: nil,
```
All default values can be 'nil' I guess.

Reply:
It should work in a way that, if `images` are not provided then, default behaviour should be taken.